### PR TITLE
Add Shields.io badges

### PR DESCRIPTION
**Pull request by celanthe**
Added lifecycle and community extension badges to README

### CODE REVIEW

**Comment by ChrisKujawa** [comment]
Hey @celanthe thanks. Is there a specific reason why some of the helm charts have different lifecycle phases and why it is not marked as stable? Maybe we can discuss this further.

**Comment by ChrisKujawa** [comment]
I discussed this with @menski and he mentioned that we don't want to mark it as stable for now, so it is fine to use https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#incubating-

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Community Extension](https://img.shields.io/badge/Community%20Extension-An%20open%20source%20community%20maintained%20project-FF4700)](https://github.com/camunda-community-hub/community)[![Lifecycle: Proof of Concept](https://img.shields.io/badge/Lifecycle-Proof%20of%20Concept-blueviolet)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#proof-of-concept-)
```
Comment (ChrisKujawa) [inline README.md]:
```suggestion
[![Community Extension](https://img.shields.io/badge/Community%20Extension-An%20open%20source%20community%20maintained%20project-FF4700)](https://github.com/camunda-community-hub/community)[![Lifecycle: Incubating](https://img.shields.io/badge/Lifecycle-Incubating-blue)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#incubating-)
```

**Review by ChrisKujawa — APPROVED** [review]

**Comment by celanthe** [comment]
Awesome, TYSM @Zelldon ! I committed your change and this should be ready to merge! :)

**Comment by ChrisKujawa** [comment]
thanks @celanthe :rocket:

**Pull request merged**